azerothcore/azerothcore-wotlk
# Furlbrow broadcasts

## Changes Proposed:
-  Adds SmartAI to Farmer Furlbrow and Verna Furlbrow to say their RP texts
-  All text values were already present but AI was missing

## Issues Addressed:

- Partially address https://github.com/azerothcore/azerothcore-wotlk/issues/11460

## SOURCE:


## Tests Performed:

- Confirmed NPCs now both say their two each texts


## How to Test the Changes:


1. Wait next to Farmer Furlbrow and Verna Furlbrow
2. Confirm each will say their two different texts

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
